src/main/java/com/mohyehia/ecommerce/aop/AppLogger.java

```diff
@@ -0,0 +1,39 @@
+package com.mohyehia.ecommerce.aop;
+
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StopWatch;
+
+import java.util.Arrays;
+
+@Component
+@Aspect
+public class AppLogger {
+    private final Log log = LogFactory.getLog(this.getClass());
+
+    @Around("execution(* com.mohyehia.ecommerce.controller..*.*(..))")
+    public Object logAround(ProceedingJoinPoint joinPoint) throws Throwable {
```

Comment:
we can refactor the function by having 2 more functions for loggingStatus & loggingExecutionTime